include/EVT/io/GPIO.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef _EVT_GPIO_
+#define _EVT_GPIO_
+
+#include <cstdint>
+
+// Forward declarations
+namespace EVT::core::IO
+{
+    // The different pins are hardware specific. Forward declaration to allow
+    // at compilation time the decision of which pins should be used.
+    enum class Pin;
+}
+
+
+namespace EVT::core::IO
+{
```

Comment:
Redundancy?
```suggestion
namespace EVT::core::IO
{
// Forward declarations:
// The different pins are hardware specific. Forward declaration to allow
// at compilation time the decision of which pins should be used.
enum class Pin;

```

Reply:
Yeah that is redundant. Originally I had that to separate the code from the forward declarations, but the command should be plenty.